### PR TITLE
Added angular-scroll

### PR DESCRIPTION
**Pull request by quanack**
Adding library for Angular Scroll located at https://github.com/oblador/angular-scroll

### CODE REVIEW

File: ajax/libs/angular-scroll/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "type": "git",
+    "url": "https://github.com/oblador/angular-scroll.git"
+  },
+  "bugs": {
+    "url": "https://github.com/oblador/angular-scroll/issues"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://git@github.com:oblador/angular-scroll.git",
```
Comment (PeterDaveHello) [inline ajax/libs/angular-scroll/package.json]: Wrong url format

Comment (quanack) [inline ajax/libs/angular-scroll/package.json]: OK, This is getting tiresome.  What is in the wrong format.  I tried this before and was told that the URL was in the wrong format and I changed it.  You need to get more descriptive on what's wrong.

Look at this https://github.com/cdnjs/cdnjs/pull/4171 where he is telling me it is the wrong format.

Comment (PeterDaveHello) [inline ajax/libs/angular-scroll/package.json]: Of course the url in line 25 is in wrong format, I think you can try it with git clone like this
`git clone git://git@github.com:oblador/angular-scroll.git`
and then you'll get the error message, I think you mixed the ssh and git protocol in the same url.

Comment (PeterDaveHello) [inline ajax/libs/angular-scroll/package.json]: http://git-scm.com/book/ch4-1.html

**Comment by PeterDaveHello** [comment]
FYI, I added in ccef47fb62b2054fb37cd3a8238778d2448bf9ff.

**Pull request closed without merging**